src/ontology/cl-edit.owl

```diff
@@ -31033,6 +31045,15 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33351116") Annotation(ob
 AnnotationAssertion(rdfs:label obo:CL_4033018 "lung megakaryocyte")
 SubClassOf(obo:CL_4033018 obo:CL_0000556)
 
+# Class: obo:CL_4033019 (blue cone bipolar cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033019 "An ON  bipolar cell type with dendrites selectively contacting S-cones. The axons of a blue cone bipolar cell terminates close to the ganglion cell layer, where it transfers the S-cone ON signal to small bistratified ganglion cells. Other targets of a blue cone bipolar cell include amacrine cells and possibly large bistratified ganglion cells.")
```

Comment:
Did not see reference to this cell type in PMID:32555229.

In text def:
- Consider using first sentence as definition and the rest as a comment. Reasoning: the first sentence contains genus and differentia which are necessary and jointly sufficient to define the cell type. The rest is "gloss", which can either remain in the definition or be relegated to a comment.
- redundant whitespace after first instance of "ON"
- subject/verb agreement issue:

Potential fix:
An ON bipolar cell type with dendrites selectively contacting S-cones. The axons of a blue cone bipolar cell **terminate** close to the ganglion cell layer, where **they transfer** the S-cone ON signal to small bistratified ganglion cells. Other targets of a blue cone bipolar cell include amacrine cells and possibly large bistratified ganglion cells.

Reply:
>Consider using first sentence as definition and the rest as a comment. Reasoning: the first sentence contains genus and differentia which are necessary and jointly sufficient to define the cell type. The rest is "gloss", which can either remain in the definition or be relegated to a comment.

Thank you for the suggestion. As it is the only bipolar cell that exclusively contact S cones, I agree that the first sentence is sufficient for the definition and the other sentences can be a comment. Thank you for the grammar and typo review!

> Is this class meant to specifically represent ON-type blue cone bipolar cell types?
[OFF-type have been reported.](https://pubmed.ncbi.nlm.nih.gov/17591913/)

Thank you for this reference. In this study the cells identified as OFF blue bipolar cells were injected, and they could only find two. In the study they did not use physiological techniques, so I would maintain blue cone bipolar cells as ON cells.

Reply:
> In this study the cells identified as OFF blue bipolar cells were injected, and they could only find two. In the study they did not use physiological techniques, so I would maintain blue cone bipolar cells as ON cells.

Bipolar cells were injected with a tracer to aid with identification. I'm unsure how this would not support the natural in vivo existence of this cell type or what a "physiological technique" is.

This study describes two OFF cone bipolar cell observed in the sample of 17 cells (whose axonal field diameters were larger than 50 μm and thus qualified as wide-field bipolar cells) that were injected.

Reply:
Although apparently derived from the abbreviation of "diffuse bipolar", DB# seems to be used as a descriptor/adjective across many of the references in this PR and not strictly as abbreviations.